prov/gni/src/gnix_cm_nic.c

```diff
@@ -531,27 +556,12 @@ int _gnix_cm_nic_alloc(struct gnix_fid_domain *domain,
 	int ret = FI_SUCCESS;
 	struct gnix_cm_nic *cm_nic = NULL;
 	gnix_hashtable_attr_t gnix_ht_attr = {0};
-	uint32_t name_type = GNIX_EPN_TYPE_UNBOUND;
 	struct gnix_nic_attr nic_attr = {0};
 
 	GNIX_TRACE(FI_LOG_EP_CTRL, "\n");
 
 	*cm_nic_ptr = NULL;
 
-	/*
-	 * if app has specified a src_addr in the info
-	 * argument and length matches that for gnix_ep_name
-	 * we must allocate a cm_nic, otherwise we first
-	 * check to see if there is a cm_nic already for this domain
-	 * and just use it.
-	 */
-
-	if (info->src_addr &&
-	    info->src_addrlen == sizeof(struct gnix_ep_name)) {
-		name_type = ((struct gnix_ep_name *)(info->src_addr))->
-								name_type;
-	}
-
```

Comment:
Perhaps this should move to a separate PR for cleanup? It could help to isolate the scalable EP changes.